data/mohit.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Mohit Singh",
+    "type": "personal",
+    "avatar": "https://avatars.githubusercontent.com/u/78977366?s=400&u=6b899ecf071531950e2b9f22d4670cb5b41726c6&v=4",
```

This is a shorter link you can use. It will automatically update as well.
```suggestion
    "avatar": "https://github.com/singhmohit14072002.png",
```